Exiled.Events/Patches/Events/Warhead/Detonated.cs

```diff
@@ -41,8 +79,8 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                     new CodeInstruction(OpCodes.Call, Method(typeof(Warhead), nameof(Warhead.OnDetonated))),
                 });
 
-            for (int z = 0; z < newInstructions.Count; z++)
```

Comment:
Changing this is a loss in performance as we're now allocating a variable for each instruction in the element when we don't need to. This is the main reason I'm requiring changes before accepting this PR